app/DoctrineMigrations/Version20170511211659.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Migrations\SkipMigrationException;
+use Doctrine\DBAL\Schema\Schema;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Increase the length of the "quote" column of "annotation" table
+ */
+class Version20170511211659 extends AbstractMigration implements ContainerAwareInterface
+{
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    public function setContainer(ContainerInterface $container = null)
+    {
+        $this->container = $container;
+    }
+
+    private function getTable($tableName)
+    {
+        return $this->container->getParameter('database_table_prefix') . $tableName;
+    }
+
+    public function up(Schema $schema)
+    {
+        $tableName = $this->getTable('annotation');
+
+        switch ($this->connection->getDatabasePlatform()->getName()) {
+            case 'sqlite':
+                $this->addSql(<<<EOD
+CREATE TEMPORARY TABLE __temp__wallabag_annotation AS 
+    SELECT id, user_id, entry_id, text, created_at, updated_at, quote, ranges 
+    FROM ${tableName}
+EOD
+                );
+                $this->addSql('DROP TABLE ' . $tableName);
+                $this->addSql(<<<EOD
+CREATE TABLE ${tableName}
+(
+    id INTEGER PRIMARY KEY NOT NULL,
+    user_id INTEGER DEFAULT NULL,
+    entry_id INTEGER DEFAULT NULL,
+    text CLOB NOT NULL,
+    created_at DATETIME NOT NULL,
+    updated_at DATETIME NOT NULL,
+    quote CLOB NOT NULL,
+    ranges CLOB NOT NULL,
+    CONSTRAINT FK_A7AED006A76ED395 FOREIGN KEY (user_id) REFERENCES wallabag_user (id),
+    CONSTRAINT FK_A7AED006BA364942 FOREIGN KEY (entry_id) REFERENCES wallabag_entry (id) ON DELETE CASCADE
+);
+CREATE INDEX IDX_A7AED006A76ED395 ON wallabag_annotation (user_id);
+CREATE INDEX IDX_A7AED006BA364942 ON wallabag_annotation (entry_id);
+EOD
+                );
+
+                $this->addSql(<<<EOD
+INSERT INTO ${tableName} (id, user_id, entry_id, text, created_at, updated_at, quote, ranges) 
+SELECT id, user_id, entry_id, text, created_at, updated_at, quote, ranges 
+FROM __temp__wallabag_annotation;
```

Comment:
Do you also do it manually ? I later realized that phpstorm could also create a migration script, but Phpstorm could not change column type and it not use exactly the same technique